tests.py

```diff
@@ -61,5 +61,11 @@ def test_simple2_multiple_signals(self):
         expected_output = 'car.stop = True\n'
         self.run_vsm('simple2', input_data, expected_output, False)
 
+    def test_delay(self):
```

Comment:
Looks great. These tests will fail for now, given the state of our parsing and processing, so can you add a skip decorator to them? https://docs.python.org/3/library/unittest.html#unittest.skip

I think for now we can make those unconditional skips. While developing the complex features for the VSM we can add a flag to the testing that enables/disables running them and we can use one of skipUnless/skipIf decorator then.